media/css/firefox/personal.less

```diff
@@ -0,0 +1,194 @@
+@import "../sandstone/lib.less";
+
+@font-face {
+    font-family: 'FiraSans Regular';
+    src: url('/media/fonts/FiraSans-Regular.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'FiraSans Bold';
+    src: url('/media/fonts/FiraSans-Bold.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'FiraSans Light';
+    src: url('/media/fonts/FiraSans-Light.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+body {
+    font-family: 'FiraSans Regular'
+}
+
+#outer-wrapper {
+    background: none;
+}
+#wrapper {
+    width: 100%;
+}
+#masthead {
+    background: #0c99d5 url(/media/img/firefox/personal/header.png) 0 0 no-repeat;
+
+    @media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
+        width: 100%;
```

Comment:
The Tabzilla tab is floating to the top right of the page, but we should try and make it consistent with other mozorg pages, and set `#masthead` to a fixed width that's centered in the page. You would need to apply the background image to a parent container in this case though.